src/App.vue

```diff
@@ -47,13 +43,13 @@ watch(isSwiping, () => {
 </script>
 
 <template>
+  <div id="modal" />
```

Comment:
After removing the redundant `app.init` there was an issue with the teleport inside `Topnav` rendering before `<div id="modal" />` which was causing an error